frontend/static/js/components/profileComponents/buttonBar.js

```diff
@@ -12,8 +13,8 @@ const buttonBar = (buttons) => {
 	const buttonBar = document.createElement('div');
 	buttonBar.classList.add('button-bar');
 
-	buttons.forEach(({ className, textContent }) => {
-		const button = Button(className, textContent);
+	buttons.forEach(({ className, textContent, handler }) => {
+		let button = Button(className, textContent, handler);
```

Comment:
Is there a reason this is let and not const? (besides the JS philosophical question of 'proper usage of cost')

Reply:
no :D I mean I was first setting the handler to the button after the initial construction in Button, but left it in the Button constuction :D will fix